pkgs/development/python-modules/cppe/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, lib, cmake, cppe, eigen
+, python, pybind11, numpy, h5py, numba, scipy, pandas, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  inherit (cppe) pname version src meta;
+
+  # The python interface requires eigen3, but builds from a checkout in tree.
+  # Using the nixpkgs version instead.
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace '"external/eigen3",' '"${eigen}/include/eigen3",'
+  '';
+
+  nativeBuildInputs = [ cmake ];
+  dontUseCmakeConfigure = true;
+
+  buildInputs = [ pybind11 eigen ];
+
+  checkInputs = [
+    numpy
+    h5py
+    numba
+    scipy
+    pandas
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "cppe" ];
+
+  # Tests that require "polarizationsolver", which is not available.
```

Comment:
Can we package that easily?

Reply:
Probably yes, I will have a look: https://gitlab.com/reinholdt/polarizationsolver
There is no release for this package, but looks easy otherwise. It is just used for the tests, although and nowhere else.

Reply:
I've added `polarizationsolver` and its dependency `fields` and no tests in `cppe` need to be disabled anymore.